app/src/main/java/org/glucosio/android/db/DatabaseHandler.java

```diff
@@ -750,30 +746,28 @@ public void editPressureReading(long oldId, PressureReading reading) {
         return idArray;
     }
 
-    public ArrayList<Integer> getMinPressureReadingAsArray() {
+    public List<Double> getMinPressureReadingAsArray() {
         List<PressureReading> readings = getPressureReadings();
-        ArrayList<Integer> readingArray = new ArrayList<Integer>();
+        ArrayList<Double> readingArray = new ArrayList<>(readings.size());
         int i;
 
         for (i = 0; i < readings.size(); i++) {
-            int reading;
             PressureReading singleReading = readings.get(i);
-            reading = singleReading.getMinReading();
+            double reading = singleReading.getMinReading();
             readingArray.add(reading);
         }
 
         return readingArray;
     }
 
-    public ArrayList<Integer> getMaxPressureReadingAsArray() {
+    public List<Double> getMaxPressureReadingAsArray() {
```

Comment:
The method name does not reflect the return type.

Reply:
Also renamed